html-forms/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <label for="email">Email </label>
+  <br><input type="email" name="email"><br>
+  <label for="select-question">Why is coding cool? </label><br>
+  <select name="Select-an-option" id="select-question">
+    <option value="Select-an-option" selected>Select an option</option>
+    <option value="Interesting">Because it's interesting</option>
+    <option value="Fun">Because it's fun</option>
+    <option value="Powerful">Because it's powerful</option>
+  </select><br>
+  <label for="language">Favorite Language (choose one)</label><br>
+  <input type="radio" name="language" id="HTML" value="HTML">HTML
+  <input type="radio" name="language" id="CSS">CSS
+  <input type="radio" name="language" id="JavaScript">JavaScript <br>
```

Comment:
you want to use a label for each of the inputs here to make sure you have the ability to click on the label and have it activate the input.